apps/zotonic_mod_zotonic_site_management/priv/skel/blog/src/SITE_sup.erl

```diff
@@ -0,0 +1,43 @@
+%% @author %%FULLNAME%%
+%% @copyright %%YEAR%% %%FULLNAME%%
+%% Generated on %%DATE%%
+%% @doc %%SITE%% root supervisor.
+
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+
+-module(%%SITE%%_sup).
+-author("%%FULLNAME%%").
+
+-behaviour(supervisor).
+
+%% API functions
+-export([ start_link/0 ]).
+
+%% Supervisor callbacks
+-export([ init/1 ]).
+
+-define(SERVER, ?MODULE).
+
+-spec start_link() -> supervisor:startlink_ret().
+start_link() ->
+    supervisor:start_link({local, ?SERVER}, ?MODULE, []).
+
+-spec init([]) -> {ok, {supervisor:sup_flags(), [supervisor:child_spec()]}}.
+init([]) ->
+    SupFlags = #{
+        strategy => one_for_all,
+        intensity => 0,
```

Comment:
This means that the site is not restarted in case of a crash. That is probably not what we want. It is nice though that an erroring site will not take down the whole server. FYI, for that problem zotonic's site supervisor had to take extra precautions.

Maybe OTP's defaults work better here... max 1 restarts in 5 seconds.

Reply:
Changed to use supervisor defaults for intensity and period in 9612ffe.